misc/python/materialize/mzcompose/__init__.py

```diff
@@ -75,8 +75,7 @@
     "enable_sink_doc_on_option": "true",
     "enable_assert_not_null": "true",
     "enable_specialized_arrangements": "true",
-    # TODO(def-,bkirwi): Reenable before this is used in production, #22042
-    # "persist_fast_path_limit": "1000",
+    "persist_fast_path_limit": "1000",
```

Comment:
Out of curiosity, what's the result of the linked issue? Can you point me at something to look at?

Reply:
The link is just to the overall testing checklist for fast-path peeks: https://github.com/MaterializeInc/materialize/issues/22042

(We have a test that complains if a TODO links to a closed issue... so IIRC the idea was that if I'd closed that issue before this was reverted, I'd get a ping to go fix it.)